pulp_ansible/app/settings.py

```diff
@@ -21,4 +21,4 @@
 }
 
 ANSIBLE_API_HOSTNAME = "http://" + socket.getfqdn()
-ANSIBLE_CONTENT_HOSTNAME = settings.CONTENT_ORIGIN + "/pulp/content"
+ANSIBLE_CONTENT_HOSTNAME = getattr(settings, "CONTENT_ORIGIN", "") + "/pulp/content"
```

Comment:
```suggestion
ANSIBLE_CONTENT_HOSTNAME = getattr(settings, "CONTENT_ORIGIN", "http://" + socket.getfqdn()) + "/pulp/content"
```

Reply:
I don't think we can do this because we can't assume the http protocol.